en/pocket_privacy_policy_eu.md

```diff
@@ -0,0 +1,124 @@
+
+## Use of Cookies and Analytics {: #cookies }
+
+### Types and Purposes
+
+We use four types of cookies and analytics tools for the following purposes:
+
+* Necessary: These cookies are necessary for our website to function. They cannot be switched off. You can set your browser to block or alert you about these cookies, but this may cause parts of the site to not work properly. These cookies do not store any personally identifiable information and unless Personalized Advertising Cookies are enabled, ads are non-personalized.
+* Preferences: We place these cookies on your device to remember your account preferences and language settings. Disabling these cookies may break some features.
+* Analytics: We place these cookies on your device to help us understand how users engage with our website and product. For example, we use Google Analytics and other tools to understand which Pocket webpages are popular. We also use pixels in our emails in order to collect usage information.
+* Personalized Advertising: Some Pocket web pages have ads. With your consent, Pocket’s ad partners will place advertising cookies on your device to personalize the ads you see here and on other websites. If you play a YouTube video on Pocket, YouTube may place cookies on your device which can be used for personalization. Our partner for personalized ads on our syndicated article pages, Freestar, has its own [Privacy Policy](https://freestar.com/privacy-policy/).
+
+### Your Choices
+
+* Cookie Preference tool: Cookie Preference tool: By default, Pocket sets necessary cookies, and disables all other types of cookies. You can manage your settings for these cookies from this Cookie Preferences tool. [Learn more](https://help.getpocket.com/article/1163-pocket-website-cookies-faq).
+* Your browser: Your browser preferences menu can also help you manage cookies and prevent cross-site tracking.
+* Google’s Analytics: By default, analytics cookies, including Google Analytics, are turned off. If you opt in to analytics cookies, Google Analytics allows us to understand how you interact with our website and measure site performance to improve your user experience.
+* Additional opt-out tools: Some third-party organizations provide ways of opting out of personalized advertising delivered by participating companies. For information about how to opt out, you can visit these organizations’ websites: [Network Advertising Initiative’s opt-out tools](https://optout.networkadvertising.org/?c=1), the [Digital Advertising Initiative’s Browser Check tool](https://optout.aboutads.info/?c=2), and the [European Interactive Digital Advertising Alliance Your Ad Choices](https://www.youronlinechoices.com/uk/your-ad-choices). We use the information we collect to count users who have visited certain pages on our website, to deliver branded services and to help determine the effectiveness of promotional or advertising campaigns.
```

Comment:
I've added "We use" in the last sentence, but I'd like to make sure all the removed text compared to the US version is intentional.

Reply:
It comes from the last paragraph of the [US version](https://github.com/mozilla/legal-docs/pull/2035/files)

> We may also use “pixel tags,” “web beacons,” “clear GIFs” or similar means (individually or collectively “Pixel Tags”) in connection with emails that we send to our users in order to collect usage data. We use Pixel Tags, other technologies and the information we collect to count users who have visited certain pages on our website, to deliver branded services and to help determine the effectiveness of promotional or advertising campaigns.

I take it you didn't cut all the text, and left that part in here. So, it should end at the link, and drop from "We use the information…)?

Reply:
Yes, thank you.